blueocean-dashboard/src/main/js/components/testing/TestCaseResultRow.jsx

```diff
@@ -22,7 +22,7 @@ export default class TestCaseResultRow extends Component {
     render() {
         const { testCase, translation, locale = 'en' } = this.props;
         const duration = TimeDuration.format(testCase.duration, translation, locale);
-        const showTestCase = testCase.errorStackTrace || testCase.errorDetails || this.stdout || this.stderr;
+        const showTestCase = testCase.errorStackTrace || testCase.errorDetails || testCase.hasStdLog;
```

Comment:
Was this just a typo caused by the recent change in case? I mean, would changing that last term to `this.stdOut || this.stdErr` have fixed it, you think?

Reply:
even if it's rare..  It's technically possible to have stderr content but no stdout. 
The java model we use make it possible.